src/plugins/irc-events/ctcp.js

```diff
@@ -21,14 +21,20 @@ module.exports = function(irc, network) {
 		chan.pushMessage(client, msg);
 	});
 
-	irc.on("ctcp request", function(data) {
+	irc.on("ctcp request", (data) => {
 		switch (data.type) {
-		case "PING":
-			var split = data.message.split(" ");
+		case "PING": {
+			const split = data.message.split(" ");
 			if (split.length === 2) {
 				irc.ctcpResponse(data.nick, "PING", split[1]);
 			}
 			break;
 		}
+		case "SOURCE": {
+			const source = "https://www.github.com/thelounge/lounge";
```

Comment:
Not what I meant, sorry. I meant that if we're not okay having `https://www.github.com/thelounge/lounge` in `package.json` and want to keep `https://www.github.com/thelounge/lounge.git` there but `https://www.github.com/thelounge/lounge` in `SOURCE`, then we can still read it from `package.json` and strip it here.

Either way, I really think it shouldn't be hardcoded here.